.dockerignore

```diff
@@ -0,0 +1,506 @@
+
+# Created by https://www.toptal.com/developers/gitignore/api/macos,windows,visualstudiocode,jetbrains+all,node,python,flask
+# Edit at https://www.toptal.com/developers/gitignore?templates=macos,windows,visualstudiocode,jetbrains+all,node,python,flask
```

Comment:
I don't think we need to include this comment in this file since this was not really generated by the mentioned page, which specifically says `gitignore` instead of `dockerignore` 😅